src/mmap.cpp

```diff
@@ -204,7 +204,7 @@ namespace {
 			file_flags(mode)
 		};
 
-		return CreateFile2(convert_to_native_path_string(name.to_string()).c_str()
+		return CreateFile2(name.data()
```

Comment:
I just noticed this. This is problematic. a `string_view` (or `wstring_view` for that matter) is not guaranteed to be 0-terminated. Just passing in the pointer to the string here is not safe, since `CreateFile2()` expects a 0-terminated string